<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Gramática Livre de Contexto</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_GLC">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>411</width>
       <height>351</height>
      </rect>
     </property>
     <property name="plainText">
      <string>P -&gt; B P1 
P1 -&gt; ; B P1 | &amp;
B -&gt; K V C
K -&gt; c K | &amp;
V -&gt; v V | &amp;
C -&gt; b C2 | C1
C1 -&gt; com C1 | &amp;
C2 -&gt; K V ; C e C1 | C e C1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>521</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Resultados</string>
    </property>
    <widget class="QTextEdit" name="textEdit_passos_interm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>521</width>
       <height>511</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Cantarell';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Sentença</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_sentence">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>411</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>f b e</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ll1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>é LL(1)?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_parser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar Parser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_anal_sintatica">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Análise Sintática</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
